awsprofile/__init__.py

```diff
@@ -50,12 +51,12 @@ def parse_args(argv=sys.argv):
         quit(1)
 
     command = argv[1:]
-    return (profile, command)
+    return (profile, command, cache)
 
 def main():
-    profile, command = parse_args()
+    profile, command, cache = parse_args()
     session = botocore.session.Session(profile=profile)
-    configure_cache(session)
+    if not cache: configure_cache(session)
```

Comment:
The choice of variable name seems confusing here, as it seems to imply the opposite.